ci/scripts/check-bazel-tags.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+untagged=$(./bazelisk.sh query "rdeps(//..., hw:verilator) except attr(tags, verilator, //...) except attr(tags, manual, //...)")
```

Comment:
* Nit: We chatted offline about piping to `wc -l`. I think it would be just as good to add a comment like, "Check that all targets that depend on //hw:verilator are tagged."

* Nit: I'd say `//hw:verilator` instead of `hw:verilator` for clarity/greppability.

* Nit: The second parameter to `attr` is a pattern, so you can combine tag parts into `except attr(tags, 'verilator|manual', //...)`.

* We don't the "manual" pattern to match something like "not-manual-at-all", so maybe we should prefix and suffix the tag patterns with `\b`.

Reply:
Per that last point, I guess you'd say `except attr(tags, '\b(verilator|manual)\b', //...)`.